packages/aws-cdk/lib/init-templates/v1/app/csharp/cdk.template.json

```diff
@@ -1,3 +1,15 @@
 {
-  "app": "dotnet run -p src/%name.PascalCased%/%name.PascalCased%.csproj"
+  "app": "dotnet run -p src/%name.PascalCased%/%name.PascalCased%.csproj",
+  "watch": {
+    "include": ["**"],
+    "exclude": [
+      "README.md",
+      "cdk*.json",
+      "src/*/obj",
+      "src/*/bin",
+      "src/*.sln",
+      "src/*/GlobalSuppressions.cs",
+      "src/*/%name.PascalCased%.csproj"
```

Comment:
Wouldn't it make more sense to just exclude all `.csproj` files? I can't imagine we would ever want to track those.

Same comment in the other C# and F# templates.